pkgs/servers/mail/mailman/package.nix

```diff
@@ -77,6 +77,12 @@ buildPythonPackage rec {
   # 'runner' scripts.
   dontWrapPythonPrograms = true;
 
+  # `setup.py check` causes a strange error.
+  # error: Command '['/nix/store/s6fgyqbk8vn1014daznm5kqx90xdn86x-python3-3.10.13/bin/python3.10', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/build/tmp9yk3_jzi', '--quiet', 'flufl.lock>=5.1']' returned non-zero exit status 1.
+  # There are so many network related errors and it is difficult to run pytestCheckHook.
+  # (1822 failed, 164 passed, 2 xfailed, 26 warnings, 11 errors)
+  doCheck = false;
```

Comment:
Doesn't this just mean we're missing checkInputs?

Reply:
Unfortunately, perhaps not. Because `flufl-lock` is already included in propagatedBuildInputs.
This sphinxcontrib_newsfeed case reported "missing docutils", but docutils was unrelated.
#263504 #264924

Reply:
others have a similar issue with flufl switching to pdm: https://github.com/iterative/dvc/issues/9654

if we drop the updates e41e58e01f93206f0e4cd4d58c5a379cc3f96d7d, a490d3a9320f799c5330d989f51c636ccc2297de and enable the mailman checks 6464cb7bf7f5aa348b1153783b71b423b7a22d55 (drop commit) everything keeps working...